rules/linux/auditd/lnx_auditd_audio_capture.yml

```
@@ -1,26 +1,32 @@
 title: Audio Capture
 id: a7af2487-9c2f-42e4-9bb9-ff961f0561d5
 status: test
-description: Detects attempts to record audio with arecord utility
+description: Detect attempts to record audio using the arecord and ecasound utilities.
 references:
     - https://linux.die.net/man/1/arecord
     - https://linuxconfig.org/how-to-test-microphone-with-audio-linux-sound-architecture-alsa
-author: 'Pawel Mazur'
+    - https://manpages.debian.org/unstable/ecasound/ecasound.1.en.html
+    - https://ecasound.seul.org/ecasound/Documentation/examples.html#fconversions
+author: Pawel Mazur, Milad Cheraghi
 date: 2021-09-04
-modified: 2022-10-09
+modified: 2025-04-26
 tags:
     - attack.collection
     - attack.t1123
 logsource:
     product: linux
     service: auditd
 detection:
-    selection:
+    selection_1:
         type: EXECVE
         a0: arecord
         a1: '-vv'
         a2: '-fdat'
-    condition: selection
+    selection_2:
```

```suggestion
    selection_syscall_memfd_create:
```
https://filippo.io/linux-syscall-table/
The syscall 319 : https://manpages.debian.org/unstable/manpages-dev/memfd_create.2.en.html